File: src/Shared/Polyrific.Catapult.Shared.Dto/Constants/JobTaskDefinitionType.cs
// Copyright (c) Polyrific, Inc 2018. All rights reserved.

namespace Polyrific.Catapult.Shared.Dto.Constants
{
    public class JobTaskDefinitionType
    {
        public const string Build = "Build";
        public const string Deploy = "Deploy";
        public const string DeployDb = "DeployDb";
        public const string Generate = "Generate";
        public const string Merge = "Merge";
        public const string Pull = "Pull";
        public const string Push = "Push";
        public const string PublishArtifact = "PublishArtifact";
        public const string Test = "Test";
        public const string DeleteRepository = "DeleteRepository";
        public const string DeleteHosting = "DeleteHosting";
        public const string CustomTask = "CustomTask";
    }
}

File: src/Web/Polyrific.Catapult.Web/ClientApp/src/app/core/enums/job-task-definition-type.ts
export enum JobTaskDefinitionType {
    Build = 'Build',
    Pull = 'Pull',
    Deploy = 'Deploy',
    DeployDb = 'DeployDb',
    Generate = 'Generate',
    Merge = 'Merge',
    Push = 'Push',
    PublishArtifact = 'PublishArtifact',
    Test = 'Test',
    DeleteRepository = 'DeleteRepository',
    DeleteHosting = 'DeleteHosting',
    CustomTask = 'CustomTask'
}
